init.lua

```diff
@@ -5,6 +5,8 @@ require('cudnn.ffi')
 local C = cudnn.C
 local ffi = require 'ffi'
 
+local thc = ffi.load("THC")
```

Comment:
I can do it. Just out of curiosity, why do you need it to be guarded. I tested my patch on linux and it works fine too.